templates/projects/tasks_browse.webpack.ejs

```diff
@@ -644,15 +644,17 @@
 </div>
 
 <!-- Project owner, subadmin that are project co-owners and admins can edit/delete task properties -->
-{% if (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin %}
+{% if current_user.id in project.owners_ids or current_user.admin %}
 <div id="context-menu" class="dropdown clearfix">
     <ul class="dropdown-menu" role="menu" aria-labelledby="dropdown-menu" style="display:block;position:absolute;">
+        {% if (current_user.subadmin and current_user.id in project.owners_ids) or current_user.admin %}
         <li><a href="#" id="edit-pri" data-value="task-priority">Edit Priority</a></li>
         <li><a href="#" id="edit-red" data-value="task-redundancy">Edit Redundancy</a></li>
-        {% if project.info.sched == 'task_queue_scheduler' or project.info.sched == 'user_pref_scheduler'  %}
+        <li><a href="#" id="del-task" data-target="#delete-tasks-modal" data-toggle="modal">Delete Task</a></li>
+        {% endif %}
+        {% if (current_user.id in project.owners_ids or current_user.admin) and (project.info.sched == 'task_queue_scheduler' or project.info.sched == 'user_pref_scheduler')  %}
```

Comment:
This condition could be optimized to remove the duplicate logic from line [647](https://github.com/bloomberg/pybossa-default-theme/pull/341/files#diff-d71135813b0ca1c5f762d349de655d16bb91bbbd5cc8728754ee542e046287a2R647):

```
{% if project.info.sched == 'task_queue_scheduler' or project.info.sched == 'user_pref_scheduler' %}
```

Reply:
Currently It is readable in one line. if you would like me to rely on reading the two if statement lines then I can modify it. I was looking to change the original design as little as possible given that this is a bug fix.

Reply:
If the logic is duplicated via the parent condition, then we could optimize per above, to reduce code error.

*Similar to the other commet, I wasn't sure if the logic is absolutely necessary here or if it is indeed a duplicate of the parent condition. Thanks.*

Reply:
okay, I will change it according to your recommendation